scripts/get_device_login_key.py

```diff
@@ -21,32 +21,47 @@
 from argparse import ArgumentParser, RawDescriptionHelpFormatter
 from pprint import PrettyPrinter
 
+from aioswitcher.bridge import (
+    SWITCHER_UDP_PORT_TYPE1,
+    SWITCHER_UDP_PORT_TYPE1_NEW_VERSION,
+    SWITCHER_UDP_PORT_TYPE2,
+    SWITCHER_UDP_PORT_TYPE2_NEW_VERSION,
+)
+
 printer = PrettyPrinter(indent=4)
 
 _examples = """example usage:
 
 python get_device_login_key.py -i "111.222.11.22" -p 10002\n
+
```

Comment:
I'm not trying to be a stickler for details, but I think this redundant line break will affect the examples output by the argument parser.

Reply:
No, it's not.

Your change: 
![image](https://github.com/TomerFi/aioswitcher/assets/28388442/c68e3e95-3bd9-46b9-ac57-8e806309142b)

The source you're basing on:
![image](https://github.com/TomerFi/aioswitcher/assets/28388442/54005ce2-d33a-466d-be39-bbcc5843a646)

Reply:
Now I see what you mean.. fixed.